src/main/java/me/snowdrop/licenses/xml/DependencyElement.java

```diff
@@ -169,4 +173,9 @@ public int hashCode() {
     public String toGavString() {
         return String.format("%s:%s:%s", groupId, artifactId, version);
     }
+
+    @XmlTransient
+    public Artifact getArtifact() {
```

Comment:
I think this method should be called `toArtifact` and instead of keeping an instance of the artifact we should create a new one using the factory and the information we have in the object (groupId, artifactId and version). I don't think we should keep artifact in this object because it's only provided in one of many constructors.